Sources/GRPC/GRPCError.swift

```diff
@@ -92,6 +92,19 @@ public enum GRPCError {
     }
   }
 
+  /// The length of the received payload was longer than is permitted.
```

Comment:
If we want to be really nice to the user, we could point them to the config struct that they need to tune.